src/uct/cuda/cuda_ipc/cuda_ipc_iface.h

```diff
@@ -14,6 +14,7 @@
 
 #include "cuda_ipc_md.h"
 #include "cuda_ipc_ep.h"
+#include "cuda_ipc_cache.h"
```

Comment:
why is this needed?

Reply:
it was done to not include cuda headers under `extern "C" {}` in test, because they use some templates inside.